src/sagarShrikhande/Test1P1String.java

```diff
@@ -0,0 +1,34 @@
+package sagarShrikhande;
+
+public class Test1P1String {
+
+	int stringOccuurrence(String input, char ch) {
+		int count = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (input.charAt(index) == ch)
+				count++;
+		}
+
+		return count;
+	}
+
+	void firstSingleOccurrence(String input) {
+		char c = ' ';
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			int count = stringOccuurrence(input, ch);
+			if (count == 1) {
```

Comment:
here you can use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop